internal/admin/exports/form.go

```diff
@@ -60,6 +61,13 @@ func (f *formData) PopulateExportConfig(ec *model.ExportConfig) error {
 		}
 	}
 	ec.IncludeTravelers = f.IncludeTravelers
+	ec.ExcludeRegions = make([]string, 0)
+	for _, s := range strings.Split(f.ExcludeRegions, "\n") {
```

Comment:
can we extract this loop out to a function? duplicate code.

Reply:
done, and added test.